docs/markdown/releasenotes.md

```diff
@@ -5,6 +5,11 @@
 ### Changed features
 
 * Added new property detect.bazel.workspace.rules to replace the now-deprecated detect.bazel.dependency.type property.
+* The download mechanism [solution_name] uses for the Black Duck Signature Scanner has changed. This will case a re-download of the Signature Scanner in the next run of [solution_name].
```

Comment:
Revise to read: "Because the Black Duck Signature Scanner uses a new [solution_name] download mechanism, the next run of [solution_name] re-downloads the Signature Scanner."

Reply:
The Black Duck Signature Scanner is not downloading Detect, which is how these changes read to me.